analytics/README.md

```diff
@@ -0,0 +1,18 @@
+# TournesolDataViz
+
+Tournesol data visualization tool is a Streamlit app to visualize [tournesol.app](https://tournesol.app/) data.
+
+
+## Run Tournesol data visualization tool
+
+Install the requirement packages:
+
+```bash
+pip install -r requirements.txt
+```
```

Comment:
Maybe add a note about using a virtualenv distinct from the one used in "backend" ?